book.md

```diff
@@ -1490,6 +1490,169 @@ now explore what we can do about these.
+    -	physically modifying the system/chip
+
+In the rest of this section, we will focus on a subset of physical attacks:
+-	Side channel information leakage
+-	Physical attack using glitches
+These two forms of attacks can be carried out using low cost hardware, and have 
+been widely demonstrated by researchers on SoCs or microcontrollers developed for 
+IoT (Internet-of-Things) applications.
+
+## Physical access side-channel attacks  
+If an attacker has physical access to a device, even without debug access, the 
+attacker can collect side channel information about the program execution on a 
+processor. If the processor is used to handle cryptographic operations, the side 
+channel information can be used to deduce the crypto key(s) or the data being 
+processed. Please note that some forms of side channel attacks (e.g. rowhammer, 
+voltjockey) do not require physical access, but those attacks are not covered 
+in this section.
+
+### How is information leaked?
+The most common physical access side channel attack method is to capture the 
+voltage and current consumption of the device during its operation. Every time 
+a flip-flop toggles, the switching activity results in a small current spike. 
+Even though there is capacitance on the power distribution connections inside 
+the chip, the toggling of registers (composed of flip-flops) still results in 
+variations in the power supply current, which can be observed easily.
+Because the connections for delivering power (at the power supply, printed 
+circuit board, on chip packages as well as on the silicon dices) also contain 
+resistance, the variation of electrical current in the chip’s power supply also 
+results in variations in the power supply voltage. Again, this can be observed 
+easily if the attacker has physical access to the device.
+If an attacker has access to data acquisition equipment that can record the 
+current/voltage/power patterns, the attacker can record the “power signature” 
+for different crypto operations, including the power signature using different 
+data inputs. By applying analysis techniques like differential power analysis, 
+the attacker could extract the information being processed.
+One additional form of side-channel leakage is electro-magnetic radiation. 
+Because the processor’s clock frequency is usually in the radio frequency (RF) 
+range, the wires and tracks on the PCB become small antenna and the ripples 
+in the processor’s voltage/current results in radio frequency signals. Although 
+the RF power radiated can be tiny, it still means that an attacker can observe 
+the side-channel leakage if he/she is in close proximity from the device and has 
+the right equipment to amplify and record the RF power signature. However, the 
+risk of such attack can be reduced by reducing the radiation energy level using:
+- Shielding around the device, including ground plate on the circuit board.
```

Comment:
This also needs a blank line before the start of the bullet point list.

Reply:
Blank lines added.